Automattic/wp-calypso
# upgrade storybook

#### Changes proposed in this Pull Request

* Upgraded Storybook from `v5.2` to `v5.3` which introduces a new configuration format 
  - [Migration Guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-52x-to-53x) 
  - [Medium Article](https://medium.com/storybookjs/declarative-storybook-configuration-49912f77b78)

#### Testing instructions



* Run `npm run components:storybook:start` and confirm the Storybook starts

Fixes #

* Sets the correct `tsconfig.json` file for `fork-ts-checker-webpack-plugin` preventing unrelated errors from the `client` from being shown
